migrating/checklists/ossm-migrating-network-policies-assembly.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-network-policies-assembly]
+= Migrating network policies from Service Mesh 2 to Service Mesh 3
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-network-policies-assembly
+
+toc::[]
+
+In {SMProductName} 2, network policies are created by default when `spec.security.manageNetworkPolicy` is set to `true` in the `ServiceMeshControlPlane` resource. During the migration to {SMProduct} 3, these policies are removed.
+
+It is recommended to recreate your network policies after you have migrated your deployment and workloads. However, for security reasons, you might need to recreate them now, and then set `spec.security.manageNetworkPolicy` to `false` as outlined in the migration checklists.
```

Comment:
Possible to avoid making a recommendation here and just explaining the options so they can decide what's best for them?
```suggestion
You can re-create your network policies after migrating your deployment and workloads. If your security policies require you to re-create them earlier, do so and then set the `spec.security.manageNetworkPolicy` field to `false` as outlined in the migration checklists.
```